daprdocs/content/en/reference/components-reference/supported-bindings/wasm.md

```diff
@@ -0,0 +1,103 @@
+---
+type: docs
+title: "Wasm"
+linkTitle: "Wasm"
+description: "Detailed documentation on the WebAssembly binding component"
+aliases:
+- "/operations/components/setup-bindings/supported-bindings/wasm/"
+---
+
+## Overview
+
+WebAssembly is a way to safely run code compiled in other languages. Runtimes
+execute WebAssembly Modules (Wasm), which are most often binaries with a `.wasm`
+extension.
+
+The Wasm Binding allows you to invoke a program compiled to wasm, by passing
+commandline args or environment variables to it, similar to how you would with
+a normal subprocess. For example, you can satisfy an invocation using Python
+even though Dapr is written in go and running on a platform that doesn't have
+Python installed!
+
+The Wasm binary must be a program compiled with the WebAssembly System
+Interface (WASI). The binary can be a program you've written such as in Go, or
+an interpreter you use to run inlined scripts, such as Python.
+
+Minimally, a user must specify a Wasm binary compiled with the canonical WASI
+version `wasi_snapshot_preview1` a.k.a. `wasip1`, often abbreviated to `wasi`.
+
+If compiling yourself in Go 1.21+, this is `GOOS=wasip1 GOARCH=wasm`, whereas
+in TinyGo, Rust and Zig, this is the target `wasm32-wasi`.
+
+You can also re-use an existing binary. For example, [Wasm Language Runtimes](https://github.com/vmware-labs/webassembly-language-runtimes)
+distributes interpreters including PHP, Python and Ruby, already compiled to
+WASI.
+
+Wasm binaries are loaded from a URL. For example, a URL "file://rewrite.wasm"
+loads "rewrite.wasm" from the current directory of the process. On Kubernetes,
+see [mounting volumes to the Dapr sidecar]({{< ref kubernetes-volume-mounts.md >}})
+to configure a filesystem mount that can contain Wasm binaries.
+
+Dapr embeds [wazero](https://wazero.io) which loads programs compiled with the
+WebAssembly System Interface (WASI), obviating platform dependencies or forking
+otherwise needed to accomplish something like this.
```

Comment:
> Dapr embeds wazero, which loads programs compiled with the WASI, obviating platform dependencies or forking otherwise needed to accomplish something like this.

This sentence is a little odd... is there a way you can reword this to make it more clear?

Reply:
sure. I'll give a go on rebase! thanks for all the help so far

Reply:
tried in https://github.com/dapr/docs/pull/3362/commits/8bcf61570e5b1a0bce76cf6835195411c647de53 lemme know

Reply:
Awesome, much clearer! Thanks!